hasher-matcher-actioner/terraform/outputs.tf

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+# These root outputs can be used as inputs for tests of delpoyed instances.
+
+output "bucket_name" {
```

Comment:
Can you describe the bucket? Would love it if the variable name gave an idea of what I'm supposed to do with the bucket.

Reply:
Followed up offline, keeping names generic for now as we use the same bucket for various different purposes. (someday it might make sense to have 'data_upload_bucket' and 'index_store_bucket' for example).